fdbsyncd/Makefile.am

```diff
@@ -0,0 +1,17 @@
+INCLUDES = -I $(top_srcdir) -I $(top_srcdir)/warmrestart   -I ../nlapi/include -I ../cfgmgr

```

Comment:
Hi @kishorekunal01  May I know `nlapi` is related to which PRs, thanks

Reply:
Yes, nlapi is in another PR. I will remove this from this PR.